src/sensor_state_data/library.py

```diff
@@ -56,3 +64,7 @@ class SensorLibrary:
         device_class=SensorDeviceClass.CO2,
         native_unit_of_measurement=Units.CONCENTRATION_PARTS_PER_MILLION,
     )
+    VOC__CONCENTRATION_PARTS_PER_MILLION = BaseSensorDescription(
```

Comment:
I know it's long, but it would be cleaner to use the full name for VOC so we are consistent here

Reply:
I noticed that the PM25 etc are missing the units, but I was afraid of changing it due to breaking things

Reply:
Let me know if I can change these too. VOC is changed

Reply:
We should add the units for PM10 etc but leave the old ones in place and add a note they are deprecated so we don't break anything.  Once they are gone from all libs we can rip them out.

Reply:
Ok, will do that after diner

Reply:
I'll merge this one now get you unblocked, since I'll be away from my desk for a bit.

 We do do the deprecation in another PR